### ISSUE TITLE
Friable oral mucosa

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11666]
Friable oral mucosa

## Proposed definition

An abnormality of the oral mucosa characterized by increased fragility, such that the tissue tears or bleeds easily with minimal mechanical trauma.

## Parent term

- `HP:0011830 Abnormal oral mucosa morphology`

## Synonyms

- Fragile oral mucosa

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
Gingival fragility HP:0034518 should be a child of this term - the gingiva is the gum tissue surrounding the teeth, and this term is more general.

**Comment by evanc1648** [comment]
Thanks @pnrobinson! That makes sense. Since the gingiva is part of the oral mucosa, gingival fragility fits neatly as a child of this term.

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001535
- label: Oral mucosal fragility
- parents: [Abnormal oral mucosa morphology (HP:0011830)](https://hpo.jax.org/app/browse/term/HP:0011830)
- pmid: PMID:41883891
- def: Increased fragility of the oral mucous membranes, such that the tissue tears or bleeds easily with minimal mechanical trauma.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11666
- synonyms: Friable oral mucosa(NONE)
